modules/jooby-couchbase/src/main/java/org/jooby/couchbase/Couchbase.java

```diff
@@ -203,7 +203,8 @@
  */
 package org.jooby.couchbase;
 
-import com.couchbase.client.deps.io.netty.util.internal.MessagePassingQueue.Supplier;
+
+import com.couchbase.client.deps.io.netty.util.internal.shaded.org.jctools.queues.MessagePassingQueue;
```

Comment:
Usage of `com.couchbase.client.deps.io.netty.util.internal.MessagePassingQueue.Supplier` is wrong, should be a `java.util.function.Supplier`

Reply:
sure, fixed in https://github.com/jooby-project/jooby/pull/1184/commits/60487b36bd90be4e16a3a1d67030891c166074c0